README.md

```diff
@@ -593,15 +593,17 @@ that are common to all storage systems are omitted for brevity.
 | **Features**                | Gaia | [Sia](https://sia.tech/) | [Storj](https://storj.io/) | [IPFS](https://ipfs.io) | [DAT](https://datproject.org) | [SSB](https://www.scuttlebutt.nz/) |
 |-----------------------------|------|--------------------------|----------------------------|-------------------------|-------------------------------|------------------------------------|
 | User controls where data is hosted           | X |   |   |   |   |   |
-| Data can be viewed in a normal Web browser   | X |   |   | X |   |   |
+| Data stored redundantly                      | optional | X | X | optional | optional | optional |
 | Data is read/write                           | X |   |   |   | X | X |
-| Data can be deleted                          | X |   |   |   | X | X |
+| Data can be deleted                          | X | X |   |   | X | X |
 | Data can be listed                           | X | X | X |   | X | X |
 | Deleted data space is reclaimed              | X | X | X | X |   |   | 
-| Data lookups have predictable performance    | X |   | X |   |   |   |
+| Data lookups have predictable performance    | X | X | X |   |   |   |
 | Writes permission can be delegated           | X |   |   |   |   |   |
 | Listing permission can be delegated          | X |   |   |   |   |   |
-| Supports multiple backends natively          | X |   | X |   |   |   |
 | Data is globally addressable                 | X | X | X | X | X |   | 
 | Needs a cryptocurrency to work               |   | X | X |   |   |   |
+| Needs FIAT currency to work                  | X |   |   |   |   |   |
 | Data is content-addressed                    |   | X | X | X | X | X |
+| Data stored geographically distributed       | optional | X | X | X | X | X |
```

Comment:
This is definitely _not_ guaranteed by IPFS -- your data is only as distributed as your pinners (and for the long tail of content, there is one pinner: you).  I don't think it's guaranteed by Sia either, unless you can demonstrate that the erasure codes actually _do_ get geo-replicated (which is basically impossible to prove -- miners can simply run many VPN endpoints in different countries, for example).  I think Storj will give you an interface for doing this, but I don't know how they enforce storage nodes to conform to it (same problem).  SSB and DAT are questionable as well -- it depends on how many replicas you can convince other people to create and persist.

Reply:
Agreed on IPFS. I know Storj is already/is going to be vetting storage nodes of a certain size with identity-tied physical contracts they have to sign. With Sia this is going to be a layer two job again, not something that happens on the protocol lever. That said, it is easy to find if someone is running a VPN; Just look at application latency and latency to first byte. This is fairly easy to filter out.

Reply:
> That said, it is easy to find if someone is running a VPN; Just look at application latency and latency to first byte. This is fairly easy to filter out.

I agree that there are heuristics that you can use to identify _likely_ VPN endpoints.  However, "likely" is un-quantifiable here.  No p2p storage system that I'm aware of (or any sort of wide-area telemetry protocol, for that matter) can guarantee with any certainty that a particular IP address corresponds to a particular server on the Internet -- not even to within a measurable error bound.

Reply:
Again, as i mentioned above already, this is true for every storage provider including Gaia. Even AWS can trick you into believing the data is stored on a server it is not. It is actually a lot easier for AWS to trick you than Sia since there is zero proof or transparency from AWS on which storage node the data is stored.

Reply:
Summarizing my comment above, I can sue AWS for tricking me.  I cannot sue anyone in the Sia network.  Moreover, Sia miners can profit by defrauding me without recourse.